### PR TITLE
fix ci for core

### PR DESCRIPTION
**Pull request by xiangyan99**

### CODE REVIEW

File: sdk/metricsadvisor/azure-ai-metricsadvisor/setup.py [hunk 1]
```diff
@@ -67,5 +67,6 @@
         "azure-core<2.0.0,>=1.23.0",
         "msrest>=0.6.21",
         'six>=1.11.0',
+        "python-dateutil>=2.8.0"
```
Comment (kristapratico) [inline sdk/metricsadvisor/azure-ai-metricsadvisor/setup.py:70]: I think this should only be a test dependency

File: sdk/core/azure-core/tests/async_tests/test_polling_async.py [hunk 2]
```diff
@@ -145,9 +145,9 @@ def deserialization_callback(response):
     assert raw_poller.polling_method() is method
     done_cb.assert_called_once_with(poller)
 
-    # Test with a basic Model
-    poller = AsyncLROPoller(client, initial_response, Model, method)
-    assert poller._polling_method._deserialization_callback == Model.deserialize
+    # # Test with a basic Model
```
Comment (scbedd) [inline sdk/core/azure-core/tests/async_tests/test_polling_async.py:148]: Did you want to clean this?

**Review by scbedd — APPROVED** [review]
Nothing here looks insane to me.

File: sdk/core/azure-core/tests/async_tests/test_polling_async.py (hunk 2 above)
Comment (xiangyan99) [reply sdk/core/azure-core/tests/async_tests/test_polling_async.py:148]: That's kind of todo. :)

**Pull request merged**